### DISCUSSION TITLE
WebAuthenticator cookies

### DISCUSSION THREAD

**Discussion by pme442** [dotMorten/WinUIEx#183] [Q&A] [answered]
Is there a way to clear the cookies?   Kind of like how the .net maui WebAuthenticator PrefersEphemeralWebBrowserSession property works.  I am using this in a .net maui app for single sign on and I can't assume that the user logging in on a device will always be the same.

**Comment by dotMorten** [comment, marked as answer]
Cookies are handled by the browser you're using to log into. WinUIEx has no say over how your browser handles this, as that is an external process.

**Reply by pme442** [reply]
OK thank you.
